.buildkite/test-e2e.yml

```diff
@@ -12,7 +12,25 @@
     - IMG=kuberay/operator:nightly make deploy
     - kubectl wait --timeout=90s --for=condition=Available=true deployment kuberay-operator
     # Run e2e tests
-    - KUBERAY_TEST_TIMEOUT_SHORT=1m KUBERAY_TEST_TIMEOUT_MEDIUM=5m KUBERAY_TEST_TIMEOUT_LONG=10m go test -timeout 60m -v ./test/e2e
+    - KUBERAY_TEST_TIMEOUT_SHORT=1m KUBERAY_TEST_TIMEOUT_MEDIUM=5m KUBERAY_TEST_TIMEOUT_LONG=10m go test -timeout 30m -v ./test/e2e/support.go ./test/e2e/raycluster_test.go ./test/e2e/rayjob_test.go ./test/e2e/rayjob_cluster_selector_test.go ./test/e2e/rayjob_lightweight_test.go ./test/e2e/rayjob_retry_test.go ./test/e2e/rayjob_suspend_test.go
+    # Printing KubeRay operator logs
+    - kubectl logs --tail -1 -l app.kubernetes.io/name=kuberay
+
+- label: 'Test E2E rayservice (nightly operator)'
+  instance_size: large
+  image: golang:1.22
+  commands:
+    - source .buildkite/setup-env.sh
+    - kind create cluster --wait 900s --config ./tests/framework/config/kind-config-buildkite.yml
+    - kubectl config set clusters.kind-kind.server https://docker:6443
+    # Build nightly KubeRay operator image
+    - pushd ray-operator
+    - IMG=kuberay/operator:nightly make docker-image
+    - kind load docker-image kuberay/operator:nightly
+    - IMG=kuberay/operator:nightly make deploy
+    - kubectl wait --timeout=90s --for=condition=Available=true deployment kuberay-operator
+    # Run e2e tests
+    - KUBERAY_TEST_TIMEOUT_SHORT=1m KUBERAY_TEST_TIMEOUT_MEDIUM=5m KUBERAY_TEST_TIMEOUT_LONG=10m go test -timeout 30m -v ./test/e2e/rayservice_ha_test.go ./test/e2e/rayservice_in_place_update_test.go ./test/e2e/support.go
```

Comment:
```suggestion
    - KUBERAY_TEST_TIMEOUT_SHORT=1m KUBERAY_TEST_TIMEOUT_MEDIUM=5m KUBERAY_TEST_TIMEOUT_LONG=10m go test -timeout 30m -v ./test/e2e2
```

Reply:
Instead of specifying each *.go file in the buildkite yaml, I prefer separate them by folders because I think it is really easy to forget or even not aware that we need to maintain the file list in buildkite yaml when adding new tests.

Reply:
how about using `e2e` and `e2e_rayservice` instead?

Reply:
Yes, that will be definitely better. Or probably `e2erayservice` which is more friendly to Go package naming.

Reply:
Thanks for the feedback. It's a good point of view. I will start to work on using `e2e` and `e2erayservice`, if there is no other consideration.